### PR TITLE
Fix #980 confusions

### PR DESCRIPTION
**Pull request by Confectrician**
THis attempts to fix confusions regarding the #980 pr.

### CODE REVIEW

**Comment by ghys** [comment]
Can you address comments I left in https://github.com/openhab/openhab-docs/commit/87a37fba0d05575f4c1c79c5f5a3be2f4b4e11a7 while you're at it? (add first-level titles to pages)

Also for "todo" sections I'd suggest a custom container:
https://vuepress.vuejs.org/guide/markdown.html#custom-containers

```
::: tip TODO
Explanation here
:::
```

**Comment by Confectrician** [comment]
> Can you address comments I left in 87a37fb while you're at it? (add first-level titles to pages)

Huh?
I have included that already.

**Comment by ghys** [comment]
😖 🤦‍♂  Sorry!

**Comment by Confectrician** [comment]
No problem^^
Preview looks good.
I will add the todo change you suggested and merge afterwards.

File: developers/index.md [hunk 1]
```diff
@@ -58,17 +58,17 @@ We have prepared some step-by-step guides for the following IDEs:
...
 <td style="width:30%">
 
-[![Visual Studio Code](ide/images/vscode.jpg)](ide/vscode.html)
+[![Visual Studio Code](./ide/images/vscode.jpg)](ide/vscode.html)
 
 </td>
 <td style="width:30%">
 
-[![Eclipse IDE](ide/images/vscode.jpg)](ide/eclipse.html)
+[![Eclipse IDE](./ide/images/vscode.jpg)](ide/eclipse.html)
```
Comment (ghys) [inline developers/index.md]:
```suggestion
[![Eclipse IDE](./ide/images/eclipse.jpg)](ide/eclipse.html)
```

**Pull request merged**